server/server.js

```diff
@@ -3,11 +3,7 @@ const express = require('express')
 
```

Comment:
Redux changes are not required here. Please remove these changes. They are not part of your ticket.